apps/portals/src/configurations/elportal/synapseConfigs/studies.ts

```diff
@@ -28,8 +28,8 @@ export const studyCardConfiguration: CardConfiguration = {
     {
       isMarkdown: false,
       baseURL: 'Explore/Projects/DetailsPage',
-      URLColumnName: 'grantNumber',
-      matchColumnName: 'grantNumber',
+      URLColumnName: 'grant',
+      matchColumnName: 'grant',
```

Comment:
Studies linking to the project details page should use the `grant` column to determine the project's unique identifier